src/libsyntax/feature_gate.rs

```diff
@@ -476,6 +476,8 @@ declare_features! (
     // 'a: { break 'a; }
     (active, label_break_value, "1.28.0", Some(48594), None),
 
+    // Integer match exhaustiveness checking
+    (active, exhaustive_integer_patterns, "1.28.0", Some(50907), None),
```

Comment:
This needs to become 1.29.0 🤷

(Also ping @eddyb for final review from non-triage.)